impl/client/src/main/javascript/web/prompting/components/DojoDateTextBoxComponent.js

```diff
@@ -201,7 +201,7 @@ define(['cdf/components/BaseComponent', "dojo/date/locale", 'dijit/form/DateText
           var dojoFormat;
           var dateFormat = this._getDateFormat();
 
-          if(dateFormat.match(/(^|(?!y).)(y{1}(?!y))/)) {
+          if(dateFormat.match(/o/)) {
```

Comment:
Why "o"? I cannot find "o" as valid symbol in either the [Dojo format documentation][1] nor the [SimpleDateFormat][2] documentation. What do I miss?

[1]: https://dojotoolkit.org/reference-guide/1.10/dojo/date/locale/format.html
[2]: https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html